common-dup/src/main/java/org/duracloud/mill/dup/DuplicationPolicy.java

```diff
@@ -41,6 +43,9 @@
 
     private List<String> spacesToIgnore = new LinkedList<>();
 
+    private static Set<DuplicationStorePolicy> EMPTY_SET =
+        Collections.synchronizedSet( new HashSet<DuplicationStorePolicy>( 0 ) );
```

Comment:
Perhaps `Collections.unmodifiableSet()` would be a better choice here? I don't see a need to synchronize on an object that's not meant to be changed.

Reply:
good catch - I intended an unmodifiableSet.